### DISCUSSION TITLE
Problem where convert to issue function does not work properly

### DISCUSSION THREAD

**Discussion by fkdl0048** [community/community#52030] [Projects and Issues] [answered]
### Select Topic Area

Question

### Body

Is it correct that the current convert to issue function is not working normally?

If so, I wonder why.

**Comment by Anmol-Baranwal** [comment]
Can you please provide more context or a specific link to the issue or repository on GitHub where you are experiencing this problem

**Reply by fkdl0048** [reply]
https://github.com/The-Day-In-1984/1984_Project/issues/28

All but this repo does not activate convert to issue.

**Comment by toddsiegel** [comment]
Are you referring to this? #51960

**Reply by fkdl0048** [reply]
Yes, that's right.. It's so sad..

**Comment by fkdl0048** [comment, marked as answer]
Thank you to the respondents.

It is currently resolved.
